SETUP.md

```diff
@@ -66,6 +95,17 @@ To perform backup, BBQ needs rights to read BigQuery data from the project which
     ```
 * (Optionally) Configure schedule time and kinds to export in [cron.yaml](./config/cron.yaml) file.
 
+### Security Layers
+
+ * **Firewall**
+   * A firewall provides identity-agnostic access control for your App Engine app.
+   Current firewall setup only allows GAE cron requests and GAE task queue requests.
+ * **IAP**
+   * Cloud [Identity-Aware Proxy](https://cloud.google.com/iap/docs/concepts-overview) (Cloud IAP) lets you manage access to GAE application.
+     Standard GAE application uses primitive roles, so it gives wider access than necessary,
+     but with IAP you can have better control of traffic in your application.  
+ * **GAE admin endpoints**
+   * Secured endpoints with admin privilege, restrict access to user which have administrator rights.
```

Comment:
>Secured endpoints with admin privilege restrict access to users with administrator rights.

Also, lets be clear what does it mean admin here. I guess this refers to login: admin which requires user to be listed in GCP projects IAMs, is that right?

Reply:
I rephrased this.